application/controllers/UserGroupController.php

```diff
@@ -100,98 +100,96 @@ public function actionIndex()
      */
     public function actionViewGroup($ugid, bool $header = false)
     {
-        if (!Permission::model()->hasGlobalPermission('usergroups', 'read')) {
+        if (empty(Yii::app()->session['loginID']) || !Permission::model()->hasGlobalPermission('usergroups', 'read')) {
```

Comment:
empty(Yii::app()->session['loginID']) then Permission::model()->hasGlobalPermission return false always.

Why add it ?